criu/config.c

```diff
@@ -517,6 +517,7 @@ int parse_options(int argc, char **argv, bool *usage_error,
 		{ "tls-key",			required_argument,	0, 1095},
 		BOOL_OPT("tls", &opts.tls),
 		{"tls-no-cn-verify",		no_argument,		&opts.tls_no_cn_verify, true},
+		BOOL_OPT("cgroup-dump-only-below-criu", &opts.cgroup_dump_only_below_criu),
```

Comment:
you need to add a description for this option into Documentation/criu.txt and criu/crtools.c